run.sh

```diff
@@ -183,6 +183,12 @@ if [ -n "$DOCKER_TIMEOUT" ]; then
     cat $DOCKER_CONF
 fi
 
+# Process Interval for Docker                                                     
+if [ -n "$DOCKER_INTERVAL" ]; then                                                
+    sed -i -e '/<Module dockerplugin>/a \'"    Interval ${DOCKER_INTERVAL}" $DOCKER_CONF
```

Comment:
You'll need to add a line similar to what is on line 181 `sed -i -e '/\bInterval\b/I c\' $DOCKER_CONF` to ensure that if there is an interval it is removed first.  This probably works for now because the default config hasn't been updated to include an Interval configuration.   Please add the extra sed line to remove any existing "Interval" configurations and test both sed statements against a config file previously containing an Interval config and one with out the interval config.